decoupled/component.js

```diff
@@ -89,7 +89,7 @@ module.exports = (function () {
     var repository = Component._repository.get(component.name);
 
     repository.find(component.type, function (data) {
-      done( Component._renderer.render(component.templatePath, data) );
+      Component._renderer.render(component.templatePath, data, done);
```

Comment:
Maybe you could extend it instead changing. I mean maybe create method `renderAsyncand` give possibility to choose how you want to render async or sync. What do you think?

Reply:
Thats a good idea. We can use ``setup`` method for it

Reply:
Done